db/schema.sql

```diff
@@ -24,6 +24,14 @@ CREATE TABLE event (
   PRIMARY KEY (id)
 );
 
+DROP DATABASE IF EXISTS questivalDB;
```

Comment:
this need to be removed and you need to merge your master branch with Claudias work, she has created all of the tables. Then just match your names up to claudias table names.